portal_pages/templates/portal_pages/case_study_page.html

```diff
@@ -2,16 +2,27 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block extra_css %}
-{% image self.hero_image width-1000 as heroimagedata %}
-<style>
-  .hero
-  {
-    height: 200px;
-    background-image: url({{ heroimagedata.url }});
-  }
-</style>
+    {% if self.hero_image %}
+        {% image self.hero_image width-1000 as heroimagedata %}
+        <style>
+          .hero
+          {
+            height: 200px;
+            background-image: url({{ heroimagedata.url }});
+          }
+        </style>
+    {% else %}
+        <style>
+          .hero
+          {
+            height: 200px;
+            background-image: url(/static/images/default_banner_image.jpg);
+          }
+        </style>
+    {% endif %}
```

Comment:
Rather than duplicating the if/else construct in a bunch of templates, a commonly shared pattern for this should go in the {% block hero-image %} portion of the base template. Any pages that need something different for the top area (such as the home page) should then just override the hero-image base template block.

To make this work, we need to be consistent on naming for the field that holds this image, it seems we have hero_image in at least one place and branding_banner in another. ALL pages that need to have the ability to specify this top image need to be defining the same field name (likely easiest done as a an abstract = True base model that all our pages of this type inherit from). 

(The index pages for case studies and marketplace entries also need to grow this new field/use this new pattern...and the base CMS page as well.)